### PR TITLE
First Microverse Project

### PR DESCRIPTION
**Pull request by AbdimulhinYussuf3675**
Practiced linters in a simple HTML project

### CODE REVIEW

**Review by GiuseppeTG — CHANGES_REQUESTED** [review]
Hi @AbdimulhinYussuf3675  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
✅ Professional Pull Request
✅ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GiuseppeTG** [comment]
- [ ] I noticed that you have a professional readme file in your main branch but not in the branch of this Pull Request. Kindly, make sure that your PR has the proper readme file.  :eyes:

Captura de Pantalla 2022-09-20 a la(s) 4 20 59 p  m

In this case, you can just copy the one you have in your main branch into this branch. For preventing this in future projects you can set up your readme just when you create the new repo, then create your branch from it so that you can have the same readme in both.  🤓 

Keep in mind that in future projects, if you don't have a propper readme file, your submission will be marked as invalid 😮‍💨

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# Hello-Micronaunts
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse
+
+
```
Comment (jerryowusu) [inline README.md]: Kindly add a detailed description of your project here. For instance you can write: 
`In this project, I set up a "Hello world" repository. No complex coding is required for this exercise. The goal here is to master all of the tools and best practices in setting up a GitHub repository` 
Your description is an extremely important aspect of your project. A well-crafted description allows you to show off your work to other developers as well as potential employers. The quality of a README description often differentiates a good project from a bad project.

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+
```
Comment (jerryowusu) [inline README.md]: Here you can add instructions on how to set up the project. For instance, you can provide the link for cloning your project, the text editor and browser to be used and how to open or start the project. You can use the image below as a guidance

Screenshot from 2022-09-20 22-06-16

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (jerryowusu) [inline README.md]: The link to the license is not working. You can fix this by creating a 'LICENSE' file (in the same location as the 'Readme.md' file), copy [this](https://github.com/jerryowusu/Web-Summit-Accra/blob/main/LICENSE) template into it and edit the year and name accordingly.

File: README.md [hunk 4]
```diff
@@ -1 +1,52 @@
...
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+
+## Authors
+
+👤 **Author1**
```
Comment (jerryowusu) [inline README.md]: Kindly replace your name with 'Author1'

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @AbdimulhinYussuf3675 , 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still a few changes to move on to the next project.

so close


 ## Highlights

-  All linters are passing :heavy_check_mark: 
- Correct GitHub flow :heavy_check_mark: 
- Nicely arranged files :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @AbdimulhinYussuf3675 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**